app/src/debug/java/de/grobox/transportr/TransportrDebugApplication.kt

```diff
@@ -19,11 +19,9 @@
 package de.grobox.transportr
 
 import android.content.Context
-import androidx.multidex.MultiDex
 
 class TransportrDebugApplication : TransportrApplication() {
     override fun attachBaseContext(base: Context) {
         super.attachBaseContext(base)
-        MultiDex.install(this)
```

Comment:
Great that we can get rid of this. Just leaving a pointer here to [the docs stating that the multidex support library is only necessary below API 21](https://developer.android.com/build/multidex).

Reply:
No sorry, it was just meant to be here for future reference. (I've looked it up and wanted to document it somewhere.)

Reply:
Perfect!